tests/test_dsp_pad_zeros.py

```diff
@@ -0,0 +1,48 @@
+import pyfar
+import numpy as np
+import pytest
+
+
+def test_zeropadding():
```

Comment:
Splitting up the test into several functions or adding some comments would be nice to directly see what is tested :)